docs/reference/search-api.md

```diff
@@ -53,16 +53,15 @@ Search for documents matching a query in the given index `<index name>`.
 
 #### Get parameters
 
-| Variable | Type | Description | Default value |
-|----------|------|-------------|---------------|
-| **query** | `String` | Query text. See the [query language doc](query-language.md) (mandatory) | |
-| **searchFields** | `[String]` | If set, specify the set of fields the search will be performed on | |
-| **startTimestamp** | `i64` | If set, restrict search to documents with a `timestamp >= start_timestamp` | |
-| **endTimestamp** | `i64` | If set, restrict search to documents with a `timestamp < end_timestamp`` | |
-| **startOffset** | `Integer` | Number of documents to skip | `0` |
-| **maxHits** | `Integer` | Maximum number of hits to return (by default 20) | `20` |
-| **format** | `String` | Response output format. `json` or `pretyjson`  | `pretyjson` |
-| **tags** | `[String]` | If set, the search is restricted to only splits having one of the tags | |
+| Variable                  | Type                 | Description                                                                                       | Default value                                                                                   |
+| ------------------------- | -------------------- | ------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| **query**                  | `String`           | Query text. See the [query language doc](query-language.md) (mandatory)                                          |                                                                                                |
+| **startTimestamp**         | `i64`    		 	    | If set, restrict search to documents with a `timestamp >= start_timestamp`                                                            |                                                                                |
+| **endTimestamp**           | `i64`       		    | If set, restrict search to documents with a `timestamp < end_timestamp``                                                            |                                                                                     |
+| **startOffset**            | `Integer`     	    | Number of documents to skip                                                                | `0`                                                                                             |
+| **maxHits**                | `Integer`          | Maximum number of hits to return (by default 20)                                                            | `20`                                                                                            |
+| **searchFields**           | `String`      		  | Fields to search on. Comma-separated list, e.g. "field1,field2" |                                                                                             |
```

Comment:
We can add the fact that if it's empty, the default_search_feilds of the index will be used.

Reply:
I don't think the default search fields are plugged in, though.

Reply:
will check that

Reply:
it's correclty plugged :)